web/ui/navbar/index.svelte

```diff
@@ -23,29 +25,23 @@
   import NavbarProgress from './progress.svelte'
   import NavbarSlow from './slow.svelte'
 
-  let isMenuOpened = $state(false)
-
-  function closeOnAnyClick(): void {
-    if (isMenuOpened) {
-      isMenuOpened = false
-      document.removeEventListener('click', closeOnAnyClick)
+  isMenuOpened.listen(isOpened => {
+    if (isOpened) {
+      setTimeout(() => {
+        document.addEventListener('click', toggleMenu)
```

Comment:
We should use here `closeMenu`. Toggle means that sometimes menu can be opened by click. It will be more clear to have explicit `closeMenu` and `openMenu` function for such cases.